Xamarin.Forms.Core/AutoCapitalization.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+    public enum AutoCapitalization
+    {
+		/// <summary>
+		/// Don't set anything on the platform component.
+		/// Currently this only affects iOS
+		/// </summary>
+		Default = 0,
+		/// <summary>
+		/// no automatic text capitalization.
+		/// </summary>
+		None = 1,
+		/// <summary>
+		/// capitalize the first character of every word.
+		/// </summary>
+		Words = 2,
+		/// <summary>
+		/// capitalize the first character of each sentence. 
+		/// </summary>
+		Sentences = 3,
+		/// <summary>
+		///  capitalize all characters. 
+		/// </summary>
+		Characters = 4
+	}
```

Comment:
what about turning a sentence to lowercase ? (`lowercase` in CSS)

Reply:
I had that thought but this issue seems to be more about the input type settings then it is about applying transformations to the text. The AutoCap settings on iOS/Android all apply to what state they place the keyboard into as the user types. UWP you can enforce rules on the box itself but that seemed to not fit the intentions of the issue. Maybe the enum needs to be changed to CapitalizationInputType?

Reply:
@StephaneDelcroix  these probably belong to  KeyboardFlags as additional flags